src/Adapter/Chain/CachePoolChain.php

```diff
@@ -24,16 +24,32 @@ class CachePoolChain implements CacheItemPoolInterface, TaggablePoolInterface
      * @type CacheItemPoolInterface[]
      */
     private $pools;
+    /** @type bool */
+    private $skipOnFailure;
 
     /**
      * @param array $pools
+     * @param bool  $skipOnFailure
      */
-    public function __construct(array $pools)
+    public function __construct(array $pools, $skipOnFailure = false)
```

Comment:
What do you think about using Symfony Option resolver? I guess we will be adding more and more construct parameters. I guess possible future options are ‘write_to_all’ and ‘defer_all_save'. 

https://github.com/php-http/plugins/blob/master/src/CachePlugin.php#L48